config/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+centos7_url:
+  - https://vault.centos.org/altarch/7.8.2003/os/ppc64le/Packages/
```

Comment:
Technically, we should probably have a [ppc64le] tag here (although the x86 CDTs dont and wont use this since they are centos6)

Reply:
Okay. So, do you want me to add another entry here for cos6 url too for x86?

Reply:
no, I don't think thats necessary

Reply:
okay.